app/models/bus_stop.rb

```diff
@@ -172,16 +49,7 @@ def route_list
   end
 
   def self.to_csv(limited_attributes: false)
-    attrs = if limited_attributes
-              LIMITED_ATTRS
-            else Hash[columns.map { |c| [c.name, c.name.humanize] }]
-            end
-    CSV.generate headers: true do |csv|
-      csv << attrs.values
-      all.each do |stop|
-        csv << attrs.keys.map { |attr| stop.send attr }
-      end
-    end
+    # TODO
```

Comment:
This is OK?

Reply:
Out of scope for this PR, I think, but should definitely be done before we merge to master, agreed.